contrib/auto-render/auto-render.js

```diff
@@ -2,6 +2,8 @@
 /* global katex */
 
 const splitAtDelimiters = require("./splitAtDelimiters");
+const Settings = require("./../../src/Settings");
+const utils = require("./../../src/utils");
```

Comment:
I'm not very familiar with ES2015. Is this the correct way to import these things from `katex`? Perhaps there's something like `require('katex.utils')` instead?